src/sagemaker_sklearn_extension/feature_extraction/sequences.py

```diff
@@ -393,30 +445,76 @@ def _hybrid_interpolation(x):
     def _extract_tsfresh_features(self, X):
         X_df = self._convert_to_df(X)
         X_df_no_nans = X_df.dropna()
-        if self.extraction_type == "minimal":
-            extraction_setting = MinimalFCParameters()
-        elif self.extraction_type == "efficient":
-            extraction_setting = EfficientFCParameters()
-        elif self.extraction_type == "all":
-            extraction_setting = ComprehensiveFCParameters()
-        else:
+        if self.extraction_type not in ["minimal", "efficient", "all"]:
             raise ValueError(
                 f"{self.extraction_type} is not a supported feature extraction option. Please choose one from "
                 f"the following options: [minimal, efficient, all]."
             )
+        min_settings = MinimalFCParameters()
         # Extract time series features from the dataframe
         # Replace any ``NaNs`` and ``infs`` in the extracted features with median/extreme values for that column
         tsfresh_features = extract_features(
             X_df_no_nans,
-            default_fc_parameters=extraction_setting,
+            default_fc_parameters=min_settings,
             column_id="id",
             column_sort="time",
             impute_function=impute,
+            n_jobs=0,
```

Comment:
I'm curious, what does `n_jobs=0` do? is it the same as `n_jobs=None`?

Reply:
n_jobs is the number of processes to use for parallelization. If not set, it uses defaults.N_PROCESSES and makes the transform jobs fail. That's why we set it to 0 (which means only use 1).